esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -100,6 +100,7 @@ short days[12] = {31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};
 
 CThorNodeGroup* CThorNodeGroupCache::readNodeGroup(const char* _groupName)
 {
+#ifndef _CONTAINERIZED
```

Comment:
minor: The whole of CThorNodeGroupCache is bare metal only (used in 1 place). I see the calling code is already conditionally compiled.

Reply:
@ghalliday - just checking, you didn't want to move this #ifndef around whole of CThorNodeGroupCache at this juncture ?